### ISSUE TITLE
Internationalization 🌍

### ISSUE THREAD

**Issue by mifi** [mifi/lossless-cut#346]
I have implemented partial support for i18n. The following is remaining:

- [ ] menus and electron.js code
- [ ] proper interpolation and pluralization, use Trans component
- [ ] Some text in Settings and App not yet i18n

For translators:

English is the base language and is only editable from code (not from weblate). Note that the english string is the base key (gettext style), so whenever I change any original text, it will flag everything for re-translation for that key.

There are a LOT of strings to translate, and I'm not going to do it, but if anyone wants to add translations, then you can go here to translate :)
https://hosted.weblate.org/projects/losslesscut/losslesscut/

All translations will be collected in [this pull request](https://github.com/mifi/lossless-cut/pull/333/files) and when there is any substantial translation there I can merge it and add support for the translated languages! 🎉

**Comment by irmatade** [comment]
Hi I added the Italian language, maybe some errors exist but I'm ready for improvement as soon as the next edition comes out.

**Comment by mifi** [comment]
I have enabled Italian and French selection now. And I have disabled system language detection so we can first review languages.

**Comment by irmatade** [comment]
Hi, some corrections have been made to the Italian language, but how can I translate the user interface translation part?

**Comment by mifi** [comment]
@irmatade  Thanks I will merge these corrections. The user interface translation is not yet implemented unfortunately, needs to be implemented before it can be translated

**Comment by GravuTrad** [comment]
Very cool. hope the total internationalization soon.
Super. J'espère une totale internationalisation rapidement.
Excellent work.

**Comment by 1024th** [comment]
I want to recommend this software to my friends who are not similar with English, so I'm eager for a total internationalization (menus and some text in Settings and App). Hoping that it's implemented soon!

**Comment by FanXiangTM** [comment]
It has not been updated for many days, and the language translation is not complete yet

**Comment by mifi** [comment]
Menus are now translatable.

**Comment by mifi** [comment]
since most things are now translatable i will close this

**Comment by dertimm** [comment]
> I have enabled Italian and French selection now. And I have disabled system language detection so we can first review languages.

Before I discovered the settings, I didn't even know about internationalization. Shouldn't the automatic speech recognition be reactivated? See #da683b3

**Comment by mifi** [comment]
You're right. I forgot about that. But now that I think about it, I'm not sure if we should auto detect language, because a lot of people are already using the app with english, and it could be surprising if the app suddenly changes language after update.
